src/frontEnd/volunteering/ViewVolunteeringOption.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Page } from '../page/Page';
+import  { LoadVolunteeringOption } from "./LoadVolunteeringOption";
+import { List } from "../../utils/list/List";
+
+
+export const ViewVolunteeringOption = ({url, organizations, navTo}) => {
+    const selectedJobID = (url.path).substring(14);
```

Comment:
Since we have a regex match expression, maybe we should use a capture group to automatically get the id out of the url in the router, passing only the ID here